trail/query/views.py

```diff
@@ -61,3 +63,16 @@ def print_results(request):
 
     return render(request, "query.html",
                   {"data": query_results, "wcsdata": wcs_list, "queryform": form, "render_table": True})
+
+@api_view(['GET'])
```

Comment:
This fails to import because the `api_view` decorator comes from `from rest_framework.decorators import api_view` which we did not import on top. Either something got broken in rebase or you accidentally overwrote the import.